esphome/esp32-d1-mini.yaml

```diff
@@ -282,6 +306,30 @@ number:
       - delay: 1s
       - switch.turn_on: mmwave_sensor
 
+# enable for HLK-LD2410, disable for SEN0395
+ld2410:
+  timeout: ${mmwave_timeout}
+  max_move_distance: ${mmwave_max_distance_move}
+  max_still_distance: ${mmwave_max_distance_still}
```

Comment:
Shouldn't we make these configurable as with the SEN0395? Or do we see it as the ld2410 platform in ESPHome should be responsible for configuration/settings?

Reply:
Haha, I totally missed that you are the author of that PR...

Reply:
Oh, one more thing - people that tested the PR said that if you put all the available sensors in the config whilst adding a web server can cause a crash in small ESPs. I suggest to remove the web server from the config to avoid such issues